src/mcts/search.cc

```diff
@@ -485,17 +485,16 @@ void SearchWorker::GatherMinibatch() {
     // Node was never visited, extending.
     ExtendNode(node);
 
-    // If node turned out to be a terminal one, no need to send to NN for
-    // evaluation.
+    // Only send non-terminal nodes to neural network
     if (!node->IsTerminal()) {
       nodes_to_process_.back().nn_queried = true;
       AddNodeToCompute(node);
     }
   }
 }
 
-// Returns node and whether it should be processed.
-// (false if it is a collision).
+// Returns node, whether it's a collision, and
```

Comment:
At that point it's still not known whether it will be evaluated, `nn_queried` will always be falst.
Line 490 is where it sets to true.

Reply:
This comment is not addressed.